### ISSUE TITLE
[Accessibility][Library] Add visible focus indicator to Component Library

### ISSUE THREAD

**Issue by gabrielwalt** [adobe/aem-core-wcm-components#1470]
On the [AEM Components Library](https://www.aemcomponents.dev/), a visual focus indicator is missing for most components:
* Breadcrumb
* Titles with links
* Images with links
* Teasers with links
* Download
* List
* Social Sharing
* Carousel
* Tabs
* Accordions
* etc…
